doc/nrf/templates/sample_README.rst

```diff
@@ -175,7 +175,7 @@ Building and running
 ********************
 
 .. note::
-   * Include the standard text for building - either ``.. include:: /includes/build_and_run.txt``, or ``.. include:: /includes/build_and_run_ns.txt`` for the non-secure build targets.
+   * Include the standard text for building - either ``.. include:: /includes/build_and_run.txt``, or ``.. include:: /includes/build_and_run_ns.txt`` for the build targets that include Non-Secure Processing Environment.
```

Comment:
```suggestion
   * Include the standard text for building - either ``.. include:: /includes/build_and_run.txt``, or ``.. include:: /includes/build_and_run_ns.txt`` for the build targets that include Non-Secure Processing Environment (NSPE).
```
?

Reply:
Rephrased to: "for the build targets that use :ref:`Cortex-M Security Extensions <app_boards_spe_nspe>`."